api-reference/openapiLangWatch.json

```diff
@@ -1,822 +1,368 @@
 {
   "openapi": "3.1.0",
+  "tags": [],
   "info": {
-    "title": "LangWatch API",
-    "version": "1.0.0"
+    "title": "Hono Documentation",
+    "description": "Development documentation",
+    "version": "0.0.0"
   },
-  "servers": [
-    {
-      "url": "https://app.langwatch.ai"
-    }
-  ],
-  "security": [
-    {
-      "api_key": []
-    }
-  ],
```

Comment:
you ended up deleting all the previous config that was here, you see including "/api/annotations" and all the other paths, unfortunately we need to do this merge by hand for now (that is, just copy and paste the new paths at the end of the array, same with schemas)